src/index.html

```diff
@@ -17,6 +17,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#logo"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          class="logo_image"
+          alt="Moyo Company Logo"
+        />
```

Comment:
HTML Formatting issue: The attributes of the `img` tag are not correctly formatted. Each attribute should start from the new line with 2-space indentation related to the tag. The tag’s closing bracket should be on the same level as opening one.